eea/climateadapt/browser/pt/aceproject_view.pt

```diff
@@ -57,6 +57,20 @@
               </div>
             </div>
 
+            <div class="macro-category" tal:condition="python:health_impacts is not None and health_impacts!='-NONSPECIFIC-'" tal:define="health_impacts context/health_impacts">
```

Comment:
wrap this in a `<div id="something">`, hide it from CSS, only show it in health observatory section

Reply:
```
#health-information { display: 'none'}
.section-observatory #health-information { display: block !important }
```